getting-started/setup-building.rst

```diff
@@ -458,6 +458,93 @@ used in ``python.sh``:
 .. _wasmtime: https://wasmtime.dev
 .. _WebAssembly: https://webassembly.org
 
+
+Emscripten
+----------
+
+Emscripten_ is a complete open source compiler toolchain. It compiles C/C++ code
+into WebAssembly_/JavaScript executables, for use in JavaScript runtimes,
+including browsers and Node.js.
+
+.. note::
+
+   The instructions below assume a Unix-based OS due to cross-compilation for
+   CPython being designed for ``./configure`` / ``make``.
+
+To build for Emscripten, you will need to cross-compile CPython. This requires a
+C compiler just like building for :ref:`Unix <unix-compiling>` as well as:
+
+1. The Emscripten compiler
+2. Node.js
```

Comment:
Numbers for ordered things, otherwise bullets:

https://asrfs.github.io/Tech-Writing-for-Techies/output/html/02-03-lists-instructions.html#_bullet_or_numbered

```suggestion
* The Emscripten compiler
* Node.js
```